integration-tests/clipper_metric_kube.py

```diff
@@ -0,0 +1,232 @@
+# This integration test is taken straight fron `kubernetes_integration_test.py
+# With the setting 1 app, 1 model, 2 replica.
+# Adding container monitoring check
```

Comment:
@simon-mo would it make sense to just add this test to the existing Kubernetes integration test, or do you think it's better to keep it separate? I'll defer to your judgment here.

Reply:
The metrics take about 3 minutes. I would keep them separate for now. But this can be a potential solution to speedup our integration test even though it makes the test less modular.